src/testclient/scripts/conf.env

```
@@ -47,7 +48,7 @@ RegionKey02[$IX,$IY]=Zone
 RegionVal02[$IX,$IY]=ap-southeast-1a
 CONN_CONFIG[$IX,$IY]=aws-ap-se-1
```

main에 수정되어 있는 내용이 반영이 안되어 있는데
conflict가 나지는 않네요? 혹시 미리 rebase가 필요할까요?
```suggestion
CONN_CONFIG[$IX,$IY]=aws-ap-southeast-1
```

---

제 로컬 환경에서 rebase 후 검색해 보니
`aws-ap-se-1` 는 사라졌고
`aws-ap-southeast-1` 로 변경되어 있습니다.

머지된 #915 의 영향으로 보입니다. 😊

제 로컬 환경에서 rebase 한 것을 force-push 하겠습니다. 😊